CONTRIBUTING.md

```diff
@@ -34,19 +34,36 @@ git checkout -b <branch name>
    
 2. Please make sure to thoroughly test your change before submitting a pull request
 
-   Please review the ["Fast, Obvious, Forgiving" experience design principles](http://www.mattermost.org/design-principles/) for Mattermost and check that your feature meets the criteria. Also, for any changes to user interface or help text, please read the changes out loud, as a quick and easy way to catch any inconsitencies
+   Please review the ["Fast, Obvious, Forgiving" experience design principles][design-principles] for Mattermost and check that your feature meets the criteria. Also, for any changes to user interface or help text, please read the changes out loud, as a quick and easy way to catch any inconsitencies
 
 
 ## Submitting a Pull Request 
 
-1. Please add yourself to the Mattermost [approved contributor list](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true) prior to submitting by completing the [contributor license agreement](http://www.mattermost.org/mattermost-contributor-agreement/). 
+1. Please add yourself to the Mattermost [approved contributor list][approved-contributor-list] prior to submitting by completing the [contributor license agreement][cla].
 
-2. When you submit your pull request please make it against `master` and include the Ticket ID at the beginning of your pull request comment, followed by a colon
+2. All contributors are required to register on the [Mattermost core team channel][core-channel-invite].
```

Comment:
Propose joining core team site as optional? 

A lot of APRs are small, and this step might slow people down unintentionally.